server/weaverbird/steps/aggregate.py

```diff
@@ -8,9 +8,11 @@
 from weaverbird.steps.base import BaseStep
 from weaverbird.types import ColumnName, PopulatedWithFieldnames, TemplatedVariable
 
-AggregateFn = Literal['avg', 'sum', 'min', 'max', 'count', 'count distinct', 'first', 'last']
+AggregateFn = Literal[
+    'avg', 'sum', 'min', 'max', 'count', 'count distinct', 'first', 'last', 'count_even_null'
```

Comment:
Would it be more coherent to name it: "count distinct including empty" ?
So it shows it's close to "count distinct", and provide a friendlier label

Reply:
:+1: